src/utils/sort.util.js

```diff
@@ -0,0 +1,6 @@
+export const SORT_OPTIONS = {
+    az: (a, b) => a.name.localeCompare(b.name),
+    za: (a, b) => b.name.localeCompare(a.name),
+    lohi: (a, b) => a.price - b.price || a.name.localeCompare(b.name),
```

Comment:
so, if there is no price in your objects(maybe somehow you failed to collect it), you'll sort products by name and then say that the order is wrong on the Inventory page?